README.md

```diff
@@ -141,7 +141,9 @@ I'm thankful to Microverse for providing a study platform which guided me throug
 This project is [MIT](https://github.com/citec-47/Leader-board/blob/leader-board/LICENSE) licensed.
 
 
+<p align="right">(<a href="#readme-top">back to the top</a>)</p>
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <p align="right">(<a href="#readme-top">back to the top</a>)</p>
```

Comment:
- [ ] Optional: I would advise you to Remove `Video explanation` section because it was not part of the requirement; Kindly remove the` Live demo` as well. You can decide to keep them, but make sure that they should not be left empty.